ng/src/web/pages/agents/component.js

```diff
@@ -51,6 +52,15 @@ class AgentComponent extends React.Component {
     gmp.agent.verify(agent).then(onVerified, onVerifyError);
   }
 
+  handleDownloadInstaller(agent) {
+    const {gmp, onInstallerDownloadError, onInstallerDownloaded} = this.props;
+
+    return gmp.agent.downloadInstaller(agent).then(response => {
+      const filename = 'agent-' + agent.id + '-installer';
```

Comment:
I'd use agent.name for a meaningful filename. Two agents can't have the same name, so filenames are unique anyway.